resources/js/manager.js

```diff
@@ -19,13 +19,15 @@ import MetricasComponent from './components/MetricasComponent';
 import DepartamentosComponent from './components/DepartamentosComponent';
 import ConfiguracionComponent from './components/ConfiguracionComponent';
 
-import ConvocatoriasCreateComponent from './components/convocatorias/ConvocatoriasCreateComponent';
+import ConvocatoriasCreateComponent from './components/ConvocatoriasCreateComponent';
+import ConvocatoriasEditComponent from './components/ConvocatoriasEditComponent';
```

Comment:
Rename ConvocatoriasEditComponent to `ConvocatoriaEditComponent `

Reply:
listo